### DISCUSSION TITLE
What features should we add?

### DISCUSSION THREAD

**Discussion by Marcoo09** [MetaLabs-inc/react-native-vimeo-iframe#57] [General]
We plan to continue working on this library in the coming weeks, so we'd love to hear your feedback.

**Comment by dels1in** [comment]
Hi,

Not sure if there is a way to do it yet,, at least I didn't find it.

But it would be cool  to add a way to change the playback rate through code

Like, setting ir through params or a prop. And that we could get the value when it changes through the "ratechange" listener

**Reply by Marcoo09** [reply]
Hi!
Currently you can add the control of speed into the [params](https://developer.vimeo.com/player/sdk/embed#:~:text=mobile%2Doptimized%20sites.-,speed,-false) 
But not change it programatically, I'll add this in the feature list and try to do it. Thanks for your suggestion.

**Comment by Poryl** [comment]
Hi guys,

I have problem with seeked video. Can you guys explain for me how can not allow user seeking the video?

Thank you!
